### PR TITLE
Fr 195 FR-309 FR-310 FR-311 mh185 positioner

### PR DESCRIPTION
**Pull request by Doomerdinger**
Waiting on [Collision fixes](https://github.com/path-robotics/motoman/pull/12) so the moveit package can be updated before the final pull in.

### CODE REVIEW

File: motoman_ma1440_d500_moveit_config/launch/move_group.launch [hunk 1]
```diff
@@ -1,7 +1,5 @@
 <launch>
 
-  <include file="$(find motoman_ma1440_d500_moveit_config)/launch/planning_context.launch" />
```
Comment (bkent324) [inline motoman_ma1440_d500_moveit_config/launch/move_group.launch]: do we need to replace this in another file or were we launching it twice?

Comment (Doomerdinger) [reply motoman_ma1440_d500_moveit_config/launch/move_group.launch]: We were launching it twice. Once here, and once in our [custom launch file](https://github.com/path-robotics/motoman/blob/42c7f8da18d4c0abac68b1956fb3792ccb3e0c17/motoman_ma1440_d500_moveit_config/launch/ma1440_d500_moveit.launch#L46) but with more arguments. 
I had considered moving the stuff from the custom launch file to here instead, just so it more closely relates to the wizard generated files, but then we'd have to add all of the custom arguments to pass into this file as well. Not opposed to moving it if that is desired.

Comment (Doomerdinger) [reply motoman_ma1440_d500_moveit_config/launch/move_group.launch]: We were launching it twice, once here and once in our [custom launch file](https://github.com/path-robotics/motoman/blob/42c7f8da18d4c0abac68b1956fb3792ccb3e0c17/motoman_ma1440_d500_moveit_config/launch/ma1440_d500_moveit.launch#L46) but with our custom arguments.
I had considered moving that to here to make it more closely resemble the auto-generated Not opposed to moving it, just a decision I made.

Comment (bkent324) [reply motoman_ma1440_d500_moveit_config/launch/move_group.launch]: works for me, I don't have a strong opinion on it one way or the other

**Pull request merged**